### ISSUE TITLE
Commandline switches are not honored by the updater

### ISSUE THREAD

**Issue by meiphoo7-Mae** [ferdium/ferdium-app#745]
### Your issue

I'm not sure if I should call this a bug or a feature request, so I'm just going to mention it without jumping to conclusions.

Recently I started testing the new experimental Wayland graphical system in Ubuntu 22 instead of the old X11.
Searching through the documentation of Electron I found out that it is possible to run Ferdium in pure Wayland mode (so not XWayland, which is a combination of X11 and Wayland).
To achieve this it is necessary to add two switches to the command that invokes Ferdium. [See this information](https://stackoverflow.com/questions/63187542/how-to-run-electron-apps-for-wayland).
This works nice and smooth and Ferdium handles the switches well. But somehow when I perform an update using the updater the commandline switches are not honored (or perhaps just ignored) and Ferdium falls back to XWayland mode.
In the past there has been a switch to start Ferdium in "hidden" mode so apparently there is (or there has been) a feature to handle switches correctly.
So once again: Ferdium itself is performing as expected but is it possible to modify the updater to respect any switches that are added to the original command?

As stated before I'm running Ubuntu 22 in Wayland mode and using the portable appimage of Ferdium.

**Comment by cino** [comment]
@meiphoo7-Mae Just to be clear, this issue is only a problem when restarting after an update? If you would open the app again from the shortcut on your system it works right? (Cause the flags should be there)

**Comment by meiphoo7-Mae** [comment]
@cino Yes, that's correct. Restarting the app once again from the shortcut after the update is what I currently do to overcome the problem.
